content-repo/extra-docs/packs/palo-alto-networks-cortex-xdr---investigation-and-response.md

```diff
@@ -394,6 +414,13 @@ The collected data generates a CSV report, including a detailed list of the disc
 The report will be sent to email addresses provided in the playbook input.
 The playbook includes an incident type with a dedicated layout to visualize the collected data.
 
+#### [Cortex XDR Lite - Incident Handling](https://xsoar.pan.dev/docs/reference/playbooks/cortex-xdr-lite---incident-handling)
+This playbook is a lite default playbook to handle XDR incidents.
```

Comment:
Can we add something like that the playbook doesn't require additional integrations to run? ( I think we should also add this to the playbook description